openerp/fields.py

```diff
@@ -1030,6 +1030,25 @@ def convert_to_cache(self, value, record, validate=True):
             return float(value or 0.0)
 
 
+class Monetary(Field):
+    type = 'monetary'
```

Comment:
Add a docstring and specify if `currency_field`is required or not (e.g. `related`?)